core/src/main/java/bisq/core/api/CoreMoneroConnectionsService.java

```diff
@@ -18,30 +22,53 @@
 import javafx.beans.property.SimpleObjectProperty;
 import javax.inject.Inject;
 import javax.inject.Singleton;
-
+import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 import monero.common.MoneroConnectionManager;
 import monero.common.MoneroConnectionManagerListener;
+import monero.common.MoneroError;
 import monero.common.MoneroRpcConnection;
-import monero.daemon.MoneroDaemon;
+import monero.common.TaskLooper;
 import monero.daemon.MoneroDaemonRpc;
+import monero.daemon.model.MoneroDaemonInfo;
 import monero.daemon.model.MoneroPeer;
 
 @Slf4j
 @Singleton
 public final class CoreMoneroConnectionsService {
 
     private static final int MIN_BROADCAST_CONNECTIONS = 0; // TODO: 0 for stagenet, 5+ for mainnet
-    private static final long DAEMON_REFRESH_PERIOD_MS = 15000L; // check connection periodically in ms
-    private static final long DAEMON_INFO_POLL_PERIOD_MS = 20000L; // collect daemon info periodically in ms
-
-    // TODO (woodser): support each network type, move to config, remove localhost authentication
-    private static final List<MoneroRpcConnection> DEFAULT_CONNECTIONS = Arrays.asList(
-            new MoneroRpcConnection("http://127.0.0.1:38081", "superuser", "abctesting123").setPriority(1), // localhost is first priority, use loopback address to match url generated by local node service
-            new MoneroRpcConnection("http://haveno.exchange:38081", "", "").setPriority(2)
-    );
+    private static final long REFRESH_PERIOD_LOCAL_MS = 5000; // refresh period when connected to local node
+    private static final long REFRESH_PERIOD_REMOTE_MS = 20000; // refresh period when connected to remote node
+
+    // default Monero nodes
+    private static final Map<BaseCurrencyNetwork, List<MoneroRpcConnection>> DEFAULT_CONNECTIONS;
+    static {
+        DEFAULT_CONNECTIONS = new HashMap<BaseCurrencyNetwork, List<MoneroRpcConnection>>();
+        DEFAULT_CONNECTIONS.put(BaseCurrencyNetwork.XMR_LOCAL, Arrays.asList(
+                new MoneroRpcConnection("http://127.0.0.1:28081").setPriority(1)
+        ));
+        DEFAULT_CONNECTIONS.put(BaseCurrencyNetwork.XMR_STAGENET, Arrays.asList(
+                new MoneroRpcConnection("http://127.0.0.1:38081").setPriority(1), // localhost is first priority, use loopback address to match url generated by local node service
+                new MoneroRpcConnection("http://stagenet.melo.tools:38081").setPriority(2),
+                new MoneroRpcConnection("http://node.sethforprivacy.com:38089").setPriority(2),
+                new MoneroRpcConnection("http://node2.sethforprivacy.com:38089").setPriority(2),
+                new MoneroRpcConnection("http://stagenet.community.rino.io:38081").setPriority(2),
```

Comment:
Do we want to add our public stagenet node? (maybe not great since it has unrestricted rpc)